src/esp/scene/GibsonSemanticScene.cpp

```diff
@@ -67,7 +79,17 @@ bool SemanticScene::loadGibsonHouse(
       scene.categories_.push_back(category);
       object->category_ = std::move(category);
     }
-    // TODO(msb) add support for aabb
+
+    const auto& jsonCenter = jsonObject["location"];
+    if (!jsonCenter.IsNull()) {
+      vec3f center = jsonToVec3f(jsonCenter);
+      const auto& jsonSize = jsonObject["size"];
+      vec3f size = vec3f::Zero();
+      if (!jsonSize.IsNull()) {
+        size = jsonToVec3f(jsonSize);
+      }
+      object->obb_ = geo::OBB(center, size, quatf::Identity());
```

Comment:
The center and size will need to transformed by the `worldRotation` parameter and the `worldRotation` will need to be to `quatf::FromTwoVectors(-vec3f::UnitZ(), esp::geo::ESP_GRAVITY)`.

Reply:
Should that be applied to the transform of the OBB instead of the center/size directly, then?

Reply:
@erikwijmans I was sure that's already done during converting `.npz` files [here](https://github.com/facebookresearch/habitat-sim/blob/998f3f7ebd91e7359ae7c585da3e507d1a509fa3/tools/npz2scn.py#L16). Let me know, if I am wrong.

Reply:
The comment says `# Transform coordinates by rotating Y-axis to Z-axis` and that seems to be what the code is doing also, so we do need to go Z to Y again. (I do agree with decision btw, it makes sense to save the meshes and annotations in the same coordinate frame, -Z gravity) CC @msbaines 

@aclegg3 The center of the OBB is assumed to already be in the correct coordinate frame and won't get applied to the center.